app/renderer/js/utils/translation-util.ts

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+import path = require('path');
+import electron = require('electron');
+import i18n = require('i18n');
+
+let instance: TranslationUtil = null;
```

Comment:
@priyank-p, @vsvipul is this correct? We have used following at other places -
```
let instance: null | ConfigUtil = null;
```

Reply:
Yes, it is! We need to do a sweep in couple of places.